vue-components/stories/Dialog.stories.ts

```diff
@@ -0,0 +1,83 @@
+import Dialog from '@/components/Dialog';
+import Button from '@/components/Button';
+import { Component } from 'vue';
+
+export default {
+	component: Dialog,
+	title: '/Dialog',
+};
+
+export function complex( args: { title: string, actions: string, dismissButton: boolean, visible: boolean, dialogText }): Component {
+	return {
+		components: { Button, Dialog },
+		props: Object.keys( args ),
+		template: `
+			<div>
+				<Button 
+					@click.native="$refs.simple.show()" 
+					variant="primary" type="progressive"
+				>
+					Open Basic Dialog
+				</Button>
+				<Dialog
+					:title="title"
+					ref="simple"
+					:actions="actions"
+					style="transform: translate(50%, 50%)"
+					@action="(_, dialog) => dialog.hide()"
+					:dismiss-button="dismissButton"
+					:visible="visible"
+				>
+					<p style="margin-bottom:10px">Hello, I am a simple dialog</p>
+					{{ dialogText }}
+				</Dialog>
+			</div>
+		`,
+	};
+}
+
+complex.args = {
+	title: 'I am a basic dialog',
+	dismissButton: 'true',
+	actions: [
+		{
+			label: 'Close me',
```

Comment:
```suggestion
			label: 'Primary action',
```

Reply:
XD sorry

Reply:
hahaha no prob! I like this Alice in Wonderland kind of copy, actually ✨